tasks/pom.xml

```diff
@@ -75,6 +76,17 @@ Copyright 2018 Google LLC
 
   <build>
     <sourceDirectory>src/main/java</sourceDirectory>
+    <plugins>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
+        <version>${maven-exec-plugin.version}</version>
```

Comment:
Does this need to be a property? I feel like there were some issues with DPEbot's auto updating properties, but I might be remembering incorrectly.

Reply:
I'm not sure. I can change it though, I was just copying from another sample in this repo that uses this plugin.